tests/artist-update.test.js

```diff
@@ -0,0 +1,33 @@
+const { expect } = require('chai')
+const request = require('supertest')
+const db = require('../src/db')
+const app = require('../src/app')
+
+describe('Update Artist', () => {
+  let artist
+  beforeEach(async () => {
+    const { rows } = await db.query('INSERT INTO Artists (name, genre) VALUES( $1, $2) RETURNING *', [
+      'Tame Impala',
+      'rock',
+    ])
+
+    artist = rows[0]
+  })
+
+  describe('PATCH /artists/{id}', () => {
+    it('updates the artist and returns the updated record', async () => {
+      const { status, body } = await request(app).patch(`/artists/${artist.id}`).send({ name: 'Frank Sinatra', genre: 'Jazz' })
+
+      expect(status).to.equal(200)
+
+      expect(body).to.deep.equal({ id: artist.id, name: 'Frank Sinatra', genre: 'Jazz' })
+    })
```

Comment:
It would be good to add more tests to check if the code works when someone sends either a name or a genre and not both